scripts/sdk-logos-cdn-check.js

```diff
@@ -0,0 +1,16 @@
+import fetch from "node-fetch";
+import { dependencies } from "../package-lock.json";
+
+async function cdnCheck () {
+  const version = dependencies["@paypal/sdk-logos"].version;
+console.log(version);
```

Comment:
Do you think we should keep these `console.log()` statements or remove them?

Reply:
I think I'm leaning towards keeping them but making them a little more verbose so it's clear to the user what info they're being given

Reply:
Just updated them but maybe the new version is too verbose.  Wouldn't mind just removing them if you think that's better.